Source/WebCore/dom/SpaceSplitString.cpp

```diff
@@ -34,12 +33,12 @@ template <typename CharacterType, typename TokenProcessor>
 static inline void tokenizeSpaceSplitString(TokenProcessor& tokenProcessor, const CharacterType* characters, unsigned length)
 {
     for (unsigned start = 0; ; ) {
-        while (start < length && isHTMLSpace(characters[start]))
+        while (start < length && isASCIIWhitespace(characters[start]))
             ++start;
         if (start >= length)
             break;
         unsigned end = start + 1;
-        while (end < length && isNotHTMLSpace(characters[end]))
+        while (end < length && isNotASCIIWhitespace(characters[end]))
```

Comment:
I considered replacing instances such as this one with `!isASCIIWhitespace` leaving `isNotASCIIWhitespace` solely for when you need a callback. We're not exactly consistent about this. Advice appreciated.

Reply:
I agree with this and think you should do what you considered. The reason `isNotASCIIWhitespace` exists is so we can use it as a predicate, which comes up mainly because of our `skip` function templates. Outside of that, I don’t think we should use it. The counter argument is that code using the “not”-named function is easier to read, but I thinkC  programmers learn to read code with `!` in it.